File: erpnext/hr/report/issue_report/issue_report.js
// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
// For license information, please see license.txt

frappe.query_reports["Issue Report"] = {
	"filters": [
                {
                        "fieldname":"from_date",
                        "label": ("From Date"),
                        "fieldtype": "Date",
                        "width": "80",
                        "reqd": 1,
			"default": frappe.datetime.year_start()
                },
                {
                        "fieldname":"to_date",
                        "label": ("To Date"),
                        "fieldtype": "Date",
                        "width": "80",
                        "reqd": 1,
			"default": frappe.datetime.year_end()
		},
		{
                        "fieldname":"module",
                        "label": ("Module"),
                        "fieldtype": "Select",
                        "options": ['', 'Human Resources', 'Accounts', 'Assets Management', 'Budget Management', 'Fleet Management', 'Material Management', 'Projects'],
                        "width": "100"
                },
		{
                        "fieldname":"issue_type",
                        "label": ("Issue Type"),
                        "fieldtype": "Select",
                        "options": ['', 'Bug', 'New Requirement'],
                        "width": "100"
                },
		{
                        "fieldname":"status",
                        "label": ("Status"),
                        "fieldtype": "Select",
                        "options": ['', 'New', 'In-discussion', 'In-development', 'On-Hold', 'Closed'],
                        "width": "100"
                }
		
	]

}

File: erpnext/hr/report/issue_report/issue_report.py
# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
import frappe

def execute(filters=None):
	columns, data = [], []
	return columns, data


def get_columns(leave_types):
        columns = [
                _("Issue List") + ":Link/Issue List:120",
                _("Issue Name") + "::150",
                _("Posting Date") +"::150",
                _("Priority") +"::160",
                _("Employment Type") +"::160",

        ]

        for leave_type in leave_types:
                columns.append(_(leave_type) + " " + _("Taken") + ":Float:160")
                columns.append(_(leave_type) + " " + _("Balance") + ":Float:160")

        return columns

def get_data(filters, leave_types):

